src/ploomber/telemetry/telemetry.py

```diff
@@ -101,7 +101,16 @@ def check_uid():
             return f"Error: Can't read UID file: {e}"
 
 
-def check_stats_file():
+def check_stats_enabled():
+    """
+    Check if the user allows us to use telemetry. In order of precedence:
+
+    1. If PLOOMBER_STATS_ENABLED defined, check its value
+    2. Otherwise use the value in stats_enabled in the config.yaml file
+    """
+    if 'PLOOMBER_STATS_ENABLED' in os.environ:
+        return os.environ['PLOOMBER_STATS_ENABLED'].lower() == 'true'
```

Comment:
We probably need to double check the original file is also set to lower() to follow the same convention

Reply:
but in YAML, there are valid booleans `false`/`true`. With env vars, they're all strings, that's why I added the lower